internal/expr/parser.go

```diff
@@ -386,16 +386,15 @@ func (p *Parser) parseColumn() (fullName, bool, error) {
 	return fullName{}, false, nil
 }
 
-func (p *Parser) parseTarget() (fullName, bool, error) {
+func (p *Parser) parseOutputType() (fullName, bool, error) {
```

Comment:
Does it make sense to rename the function above to the new convention as well?

Reply:
Changed.